CHANGELOG.md

```diff
@@ -3,6 +3,117 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.6.0
+
+_Mar 3, 2022_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📦 Use the bundling scripts as the packages published by the [https://github.com/mui/material-ui](material-ui) repository (#3965) @flaviendelangle
```

Comment:
I'm not sure if the users care enough for this to be highlight?

Reply:
Maybe it's reassuring. It will work the same way that the other library they are using.

But I agree the most important part is in the explanation below. Maybe for the tweet, we should only talk about the tree shaking

Reply:
Already asked for the banner

Reply:
```suggestion
- 📦 Use the same bundling scripts as the packages published by the [https://github.com/mui/material-ui](material-ui) repository (#3965) @flaviendelangle
```

Reply:
I could be wrong, but I think it might be more grammatically correct to say that the package is published _in_ the repository, _by_ the maintainers of that repo.

Reply:
What about a more straightforward?

```suggestion
- 📦 Use the bundling scripts as in [material-ui](https://github.com/mui/material-ui) repository (#3965) @flaviendelangle
```